### ISSUE TITLE
bedtools misdetections

### ISSUE THREAD

**Issue by bernt-matthias** [aCLImatise/CliHelpParser#36]
```
Tool:    bedtools random (aka randomBed)
Version: v2.26.0
Summary: Generate random intervals among a genome.

Usage:   bedtools random [OPTIONS] -g <genome>

Options: 
	-l	The length of the intervals to generate.
		- Default = 100.
		- (INTEGER)

	-n	The number of intervals to generate.
		- Default = 1,000,000.
		- (INTEGER)

```

these two options are currently detected as CliBoolean Flag.

**Comment by multimeric** [comment]
Great, thanks! I'll update the parser to deal with this.

**Comment by bernt-matthias** [comment]
Also the `-g` flag is undetected... but I guess this is difficult and it might be better to file a complaint at the bedtools repo

**Comment by multimeric** [comment]
Hmm, no actually it should be detected. I have a usage parser as well as a flag parser. So it is a bug.

**Comment by multimeric** [comment]
Feel free to PR this as a new test case. For example: https://github.com/aCLImatise/CliHelpParser/blob/017bfe01dab6eb133deaf6f64060277233dfa200/test/util.py#L42-L49

**Comment by bernt-matthias** [comment]
Actually I was wrong that -g is undetected. Sorry. It is detected as text parameter which is probably the best the parser can achieve.

**Comment by multimeric** [comment]
Unfortunately yes.

**Comment by multimeric** [comment]
This is fixed in #52: https://github.com/aCLImatise/CliHelpParser/blob/8f7ea4099b304605817a8400411e53e367e2589c/test/util.py#L320-L329.
